server/routes/index.js

```diff
@@ -54,11 +54,6 @@ r.get('/:preview(preview)?/exhibitions/:id', renderExhibition);
 r.get('/eventbrite-event-embed/:id', renderEventbriteEmbed);
 r.get('/series/(W):id', renderSeries);
 
-// Vanity URLs - should redirect
-r.get('/graphicdesign', async (ctx, next) => {
```

Comment:
So will `/graphicdesign` no longer redirect?

Reply:
The redirect will be setup in Drupal... negating the need for it to be in this app.

Reply:
Got it